README.md

```diff
@@ -16,6 +16,7 @@ To reproduce the results of the analysis, the modules should be run in order.
 | 1.initial-z-sweep | Determine a set of optimal hyperparameters for Tybalt and ADAGE models across a representative range of z dimensionality |
 | 2.ensemble-z-analysis | Train various algorithms to compress gene expression data across a large range of z dimensions |
 | 3.build-hetnets | Download, process, and integrate various curated gene sets into a single heterogeneous network |
+| 4.analyze-components | Visualize the reconstruction and sample correlation results of the ensemble z analysis |
```

Comment:
this README.md could benefit from some more detailed index/list of some subdirectory notebooks/scripts. for example: 

| 0.expression-download | Download and process gene expression data to run through pipeline |
| [0.1.A.download-pancanatlas-data.ipynb](hyperlink)|some description|
| [0.1.B...](some other hyperlink)|some desc|
| 1.initial-z-sweep | Determine a set of optimal hyperparameters for Tybalt and ADAGE models across a representative range of z dimensionality |
| [1.A.visualize-parameter-sweep.ipynb](hyperlink)|some desc|